app/views/accounts/_phone.html.erb

```diff
@@ -3,17 +3,15 @@
     <h2 class="grid-col-fill margin-y-0 padding-right-2">
       <%= t('account.index.phone') %>
     </h2>
-    <div class="grid-col-auto">
-      <% if EmailPolicy.new(current_user).can_add_email? %>
-        <%= link_to(
-              prefix_with_plus(t('account.index.phone_add')),
-              add_phone_path,
-              class: 'account-action-button',
-            ) %>
+  </div>
+  <div class="grid-row margin-bottom-1 margin-top-0">
+    <div class="grid-col">
+      <a id="phones"></a>
```

Comment:
This may (hopefully) trigger a build failure, since it's not a valid link. I think we could add the ID to the alert instead?

You should be able to pass HTML attributes to the constructor below:

```rb
<%= render AlertComponent.new(type: :error, id: 'phones', message: flash[:phone_error]) %>
```